<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenLock</class>
 <widget class="QWidget" name="ScreenLock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="ScreenLockPictureLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>900</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 0, 0);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TimeLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>611</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 25 72pt &quot;微软雅黑 Light&quot;;</string>
   </property>
   <property name="text">
    <string>14:16</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="DateLabel">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>720</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 25 36pt &quot;微软雅黑 Light&quot;;</string>
   </property>
   <property name="text">
    <string>4月22日, 星期日</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>507</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(234, 234, 234);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>500</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>480</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 25 15pt &quot;微软雅黑 Light&quot;;</string>
   </property>
   <property name="text">
    <string>密码错误</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="verifyButton">
   <property name="geometry">
    <rect>
     <x>736</x>
     <y>530</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:none;
background-color:rgb(188, 188, 188);
color:rgb(255, 255, 255);
font: 25 15pt &quot;微软雅黑 Light&quot;;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="AI.qrc"/>
 </resources>
 <connections/>
</ui>
